packages/http-server-csharp/src/boilerplate.ts

```diff
@@ -51,14 +51,23 @@ export function getSerializationSourceFiles(
       emitter: emitter,
       getContents: getNumericArrayConstraintConverter,
     }),
+    new LibrarySourceFile({
+      filename: "IJsonSerializationProvider.cs",
+      emitter: emitter,
+      getContents: getJsonProviderInterface,
+    }),
+    new LibrarySourceFile({
+      filename: "JsonSerializationProvider.cs",
+      emitter: emitter,
+      getContents: getJsonProvider,
+    }),
   );
   return sourceFiles;
 }
 
 function getNumericConstraintConverter(): string {
-  return `// Copyright (c) Microsoft Corporation. All rights reserved.
-  // Licensed under the MIT License.
-  // <auto-generated />
+  return `// Generated by @typespec/http-server-csharp
+// <auto-generated />
```

Comment:
should we consider a const like `fileHeader` to store such constant information so that we do not have to manually change them one by one?

Reply:
Yes, there is a current constant for licenseHeader, which is essentially that